README.md

```diff
@@ -0,0 +1,101 @@
+# Operator-custom metrics
+This library assists in collecting operator custom-metrics from operators and exposing them to prometheus instance.
+
+## Prerequisites
+In order to use the functions provided by this library download the package inside the `vendor` directory of your operator code.
+The following command can be used:
+
+```bash
+go get github.com/openshift/operator-custom-metrics
+```
+
+## Using operator-custom metrics library
+
+### Description
+The library is designed to register the metrics provided by the user to prometheus. The user can provide the **metrics configuration** based on which the metrics are collected.
+The following functions are performed by the library:
+1. Register custom metrics provided by the user to prometheus.
```

Comment:
Rather than saying `custom metrics provided by the user` again and again, let's start the `README` by highlighting how the user must define custom metrics and then just refer to them as custom-metrics throughout the document.